lib/entities/release.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import copy from 'fast-copy'
+import {
+  BaseCollection,
```

Comment:
Bonus points for sorting (I'm going to have to start paying out these bonus points someday)

Reply:
I accept bitcoins :trollface:

Reply:
no sorting means no bitcoins (or other coins) for you!